bin/VolcanoPlots_script.R

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env Rscript
+'
+Usage:
+  VolcanoPlots_script.R --de_res=<de_res> --goi=<goi> --prefix=<prefix> [options]
+
+Mandatory arguments:
+  --de_res=<de_res>           TopTable from DESeq2 in TSV format
+  --goi=<goi>                 Genes of interest in txt format
+  --prefix=<prefix>           Prefix for output filenames
+
+Optional arguments:
+  --pCutoff=<pCutoff>         Cut-off for statistical significance [default: 0.05]
+  --FCcutoff=<FCcutoff>       Cut-off for absolute log2 fold-change [default: 2]
+  --results_dir=<dir>         Output directory [default: ./]
+' -> doc
+
+library(conflicted)
+library(docopt)
+arguments <- docopt(doc, version = "0.1")
+print(arguments)
+
+library(readr)
+library(dplyr)
+library(EnhancedVolcano)
+
+
+# Load parameters
+de_res <- read_tsv(arguments$de_res)
+goi <- read_lines(arguments$goi, skip = 1)
```

Comment:
It might make sense to remove "skip = 1" as there will anyway not be a gene matching the column name "gene_name". In addition, it will not drop the first gene entry in case that no column name is provided in the txt file.

Reply:
sounds reasonable!